config/services.yaml

```diff
@@ -125,6 +125,11 @@ services:
             string $passwordExpiryWarn: "%env(ACCOUNT_PASSWORD_EXPIRATION_WARN)%"
             string $hostName: "%env(HOSTNAME)%"
 
+    App\Command\PelagosCheckForMissingFilesCommand:
+        bind:
+            string $dataStoreDirectory: "%env(DATA_STORE_DIRECTORY)%"
+            $datastoreFlysystem: "@oneup_flysystem.datastore_flysystem_filesystem"
```

Comment:
We should inject src/Util/Datastore, we shouldn't be using it directly either from a controller or command